test/test_utils/__init__.py

```diff
@@ -104,6 +104,7 @@ def is_image_incompatible_with_instance_type(image_uri, ec2_instance_type):
     Check for all compatibility issues between DLC Image Types and EC2 Instance Types.
     Currently configured to fail on the following checks:
         1. p4d.24xlarge instance type is used with a cuda<11.0 image
+        2. p2.8xlarge instance type is used with a cuda=11.0 image for MXNET framework
```

Comment:
Explicitly disabling for MXNET until we have confirmation on Pytorch and Tensorflow.